combined updates (#1) (#1)

sambacha opened the pull request:
Co-authored-by: cl <chencassc@gmail.com>

sambacha committed:
combined updates (Consensys#1) (#1)

Co-authored-by: cl <chencassc@gmail.com>

sambacha committed:
feat(lint): fixes

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/Consensys/python-solidity-parser?pullRequest=32) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/Consensys/python-solidity-parser?pullRequest=32) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/Consensys/python-solidity-parser?pullRequest=32) it.</sub>

sambacha committed:
build(scripts): setup build scripts

sambacha committed:
ci(build): use start script

sambacha committed:
ci(build): workflow fixes

sambacha committed:
fix(build): ensure submodule check

sambacha committed:
fix(submodules): anltr4

sambacha committed:
Add 'solidity-antlr4/' from commit '0eb1244ae37057ba3f434681aa35d2da08b0d633'

git-subtree-dir: solidity-antlr4
git-subtree-mainline: 980ccd59d90ef975203deb2a427e5843d412e44c
git-subtree-split: 0eb1244ae37057ba3f434681aa35d2da08b0d633
